data-model/fdo-type/term/term.json

```diff
@@ -0,0 +1,184 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://schemas.dissco.tech/schemas/fdo-type/term/0.1.0/term.json",
+  "title": "DiSSCo openDS Term Schema",
+  "description": "A schema for defining standard terms used in openDS.",
+  "type": "object",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The unique identifier (handle) of the term object",
+      "pattern": "^https://hdl\\.handle\\.net/20\\.500\\.1025/([^/]+)$",
+      "examples": [
+        "https://hdl.handle.net/20.500.1025/DigitalSpecimen"
+      ]
+    },
+    "@type": {
+      "type": "string",
+      "const": "ods:Term",
+      "description": "The type of the digital object, in this case a ods:Term"
+    },
+ "ods:fdoType": {
+      "type": "string",
+      "description": "The Handle to the FDO type of the object",
+      "pattern": "^https:\/\/hdl\\.handle\\.net\/[\\w.]+\/(.){3}-(.){3}-(.){3}",
+      "examples": [
+        "https://hdl.handle.net/21.T11148/2ac65a933b7a0361b651"
```

Comment:
This is still the VirtualCollection, better to point to the FDOType for the Term here